cmd/distribution.go

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2020 SIGHUP s.r.l All rights reserved.
```

Comment:
2022 
> Captain: what a year, huh'?
> Ttintin: Captain, it's 2022

Reply:
I'll open another PR for the cosmetic issues